oauthenticator/tests/test_globus.py

```diff
@@ -0,0 +1,35 @@
+from pytest import fixture, mark
+
+from ..globus import GlobusOAuthenticator
+
+from .mocks import setup_oauth_mock, no_code_test
+
+
+def user_model(username):
+    """Return a user model"""
+    return {
+        'login': username,
+    }
+
+@fixture
+def globus_client(client):
+    setup_oauth_mock(client,
+        host=['github.com', 'api.github.com'],
+        access_token_path='/login/oauth/access_token',
+        user_path='/user',
```

Comment:
Globus SDK generates those internally. I can construct them, but they might break down the road. Should I just invoke the Globus sdk here?